treccovid_semantic_search/index.json

```diff
@@ -0,0 +1,30 @@
+{
+  "settings": {
+    "index.number_of_shards": {{number_of_shards | default(1)}},
+    "index.number_of_replicas": {{number_of_replicas | default(0)}},
+    "index.queries.cache.enabled": {{query_cache_enabled | default(false) | tojson}},
+    "index.requests.cache.enable": {{requests_cache_enabled | default(false) | tojson}},
+    "index.knn": true,
+    "default_pipeline": "nlp-ingest-pipeline"
+  },
+  "mappings": {
+    "properties": {
+      "id": {
+        "type": "text"
+      },
+      "passage_embedding": {
+        "type": "knn_vector",
+        "dimension": {{dimensions | default(768)}},
+        "method": {
+          "engine": "{{engine | default('lucene')}}",
```

Comment:
why lucene is default? why not set only engine if it is defined ?

Reply:
Done, no default engine.